README.md

```diff
@@ -2,7 +2,51 @@
 
 # Professional, personnal action-reaction manager
 
-## Introduction
+## What is Aeris
+
+Aeris is an Action-Reaction system manager. It lets its users create *Pipelines*.
+
+Pipelines are triggered by an *Action* (for example, when a PR is open on some GitHub repository), and perfoms *Reactions* (for example, play a given song on Spotify).
+
+To manage your Pipeline, you can use the Android App, as well as the Web client.
+
+Aeris supports a variety of service, on which Pipelines are perfomed upon:
+
+- GitHub
+- Spotify
+- Youtube
+- Discord
+- Gmail
+- Twitter
+- Even more services in the future :smile:
+
+## What do I need to use Aeris?
+
+Make sure the following softwares are installed on your machine:
+
+- Docker, (and Docker-Compose)
+
+To setup Aeris, you need to provide the building system some information:
+
+- Client ID and Client Secrets for EACH services (fill the `.env.example`provided at the root of the repository)
+- A Host name, to let clients know how to call the API (usually localhost:8080)
+
+## How to install it ?
+
+To install Aeris, run the following commands at the root of the repository:
+
+- `docker-compose -f docker-compose.yml build` (grab a snack, it might take some time)
+- `docker-compose -f docker-compose.yml up`
+
+## How to use it ?
+
+The Aeris server is accessible on the host's 8080 port
+
+You can access the Web client through port 8081
```

Comment:
```suggestion
You can access the Web client through `localhost`
```

Reply:
This is not compliant to the subject: https://intra.epitech.eu/module/2021/B-YEP-500/NAN-5-1/acti-480798/project/file/B-YEP-500_area.pdf